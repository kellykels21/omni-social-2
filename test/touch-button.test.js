import React from "react";
import renderer from "react-test-renderer";

import TouchButton from "@components/atoms/touch-button";

describe("<TouchButton />", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <TouchButton
          onPress={() => {
            console.log("Pressed the button");
          }}
          title="Click Me!"
          color="red"
          height={50}
          width={200}
        ></TouchButton>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
