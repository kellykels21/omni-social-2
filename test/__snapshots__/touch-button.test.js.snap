// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TouchButton /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "red",
        "height": 50,
        "justifyContent": "center",
        "opacity": 1,
        "width": 200,
      }
    }
  >
    <Text
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      Click Me!
    </Text>
  </View>
</View>
`;
